pages/create-video/remotion/MyComp/Main.tsx

```diff
@@ -0,0 +1,104 @@
+// import { z } from "zod";
+import {
+  Sequence,
+  // spring,
+  // useCurrentFrame,
+} from "remotion";
+// import { CompositionProps } from "../../types/constants";
+import React from "react";
+// import { TextFade } from "./TextFade";
+// import { IssueTags } from "../../../../components/IssueTags/IssueTags";
+
+const tag: React.CSSProperties = {
+  width: "140pt",
+  height: "40pt",
+  padding: "24pt 14pt",
+  borderRadius: "32pt",
+  backgroundColor: "#003C55",
+  fontFamily: "proxima_nova",
+  fontSize: "2.25rem",
+  color: "white",
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+};
+
+const bigTag: React.CSSProperties = {
+  width: "48rem", // 270pt
+  height: "12rem", // 68pt
+  padding: "1.25rem 1.875rem", // 20pt 30pt
+  borderRadius: "3.75rem",
+  border: "4px solid #FFB45C",
+  backgroundColor: "rgba(255, 180, 92, 0.1)", // 10% opacity
+  fontFamily: "commuter_sans",
+  fontSize: "4rem",
+  color: "white",
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  boxShadow: "0px 0px 5px rgba(0, 0, 0, 0.3)",
+};
```

Comment:
Lets create a .MyComp.module.scss file to define these styles (and the inline ones) and import the module (see other existing components for reference as this is the pattern through the codebase.  `MyComp` also needs a more descriptive name

Reply:
Got it!